RELEASE_NOTES.md

```diff
@@ -129,10 +129,106 @@ include the new sigmoid module.  Options are now as follows:
   color harmony -- think of them like crop composition guides, but for
   colors.
 
-- Many modules have had code cleanup and performance improvements.
-  All SSE2 code paths have been removed (the optimized parallel code
-  generated by the compiler is faster) or code optimized in the
-  following modules. This has led to speed gains of 5-25%:
+- A global <kbd>right-click-and-drag</kbd> operation has been added to
+  allow image rotation to be corrected without first opening the
+  rotate and perspective module. This operation can be used as long as
+  the currently-focused module does not already use that shortcut for
+  another purpose.
+
+- Drawn mask functionality has been completely overhauled with a large number
+  of changes and fixes:
+
+  - The "brush smoothing" and "pen pressure" options have been moved
+    from the global preferences dialog to a new collapsible "properties"
+    section in the mask manager, so that they can be changed while
+    drawing and can be assigned shortcuts.
+  
+  - It is now possible to edit drawn mask shape size/feather/hardness
```

Comment:
The first line suggests this is new but this already existed in 4.2. The new bit is that they were changed to log scale and work while drawing.

Reply:
resolved in the latest commit